Lib/idlelib/calltips.py

```diff
@@ -139,21 +139,32 @@ def get_argspec(ob):
         ob_call = ob.__call__
     except BaseException:
         return argspec
-    if isinstance(ob, type):
-        fob = ob.__init__
```

Comment:
Why remove this?  It worked before.  Does signature() do this automatically?  Such a change should be explained on the issue with examples, both for Python and C coded classed.  And yes, I wish I had added comments here or in test file or, by reference, an issue, when I revised this and and added test file.